### PR TITLE
add some tests, mainly paths for exception.

### PR DESCRIPTION
**Pull request by IgnorantCoder**
relates to #27

### CODE REVIEW

File: src/decode.rs [hunk 1]
```diff
@@ -77,4 +77,68 @@ mod tests {
         assert_eq!(super::split_jwt(jwt)?, (raw_verify_target, raw_signature));
         Ok(())
     }
+
+    #[test]
```
Comment (megumish) [inline src/decode.rs]: It can be substituted assert (true) and # [should_panic] instead of assert (false)

File: src/decode.rs [hunk 2]
```diff
@@ -77,4 +77,68 @@ mod tests {
...
+                    super::ErrorKind::WrongToken => {
+                        //OK
+                    }
+                    _ => assert!(false),
+                };
+            }
+        };
+    }
+
+    #[test]
```
Comment (megumish) [inline src/decode.rs]: It can be substituted too.

File: src/decode.rs (hunk 1 above)
Comment (megumish) [reply src/decode.rs]: Oops, Does not it change much...?

Comment (IgnorantCoder) [reply src/decode.rs]: I want to check exception kind as expected.
Change to more simple one.

**Pull request merged**